XML/Other/ReplaceableTestParameters.xml

```diff
@@ -713,6 +713,10 @@ Scenario 2 : You need to run all the performance tests quickly.
 		<ReplaceThis>ENABLE_NETWORK_WORKLOAD</ReplaceThis>
 		<ReplaceWith>yes</ReplaceWith>
 	</Parameter>
+	<Parameter>
```

Comment:
why not set the value directly in the test case's xml file?

Reply:
fixed it.